templates/myaccount/form-add-payment-method.php

```diff
@@ -55,4 +55,6 @@
 			</div>
 		</div>
 	</form>
+<?php else : ?>
+	<p class="woocommerce-notice woocommerce-notice--info woocommerce-info"><?php esc_html_e( 'Sorry, it seems that there are no payment methods which support adding a new payment method. Please contact us if you require assistance or wish to make alternate arrangements.', 'woocommerce' ); ?></p>
```

Comment:
> Sorry, it seems that there are no payment methods which support adding a new payment method. Please contact us if you require assistance or wish to make alternate arrangements.

That sentence 😅 

Since this page is usually inaccessible (no buttons), let's simplify to just:

> New payment methods can only be added during checkout. Please contact us if you require assistance.

OR if we can find a way to redirect back to account, even better. This shouldn't be seen.